src/peer_digest.cc

```diff
@@ -289,7 +240,7 @@ peerDigestCheck(void *data)
 static void
 peerDigestRequest(PeerDigest * pd)
 {
-    CachePeer *p = pd->peer;
+    const auto p = pd->peer.get(); // TODO: Replace with a reference.
```

Comment:
FWIW, I agree that this TODO should be left for a follow up polishing PR to decrease noise in this bug-fixing PR.